svo (install me in module manager).xml

```diff
@@ -7187,8 +7190,7 @@ end</script>
 				<name>svo.classchange</name>
 				<packageName></packageName>
 				<script>function svo.classchange(override)
-
-  local newclass = (override ~= "gmcp.Char.Status") and override or gmcp.Char.Status.class
+  local newclass = (override ~= gmcp.Char.Status.class) and override or gmcp.Char.Status.class
```

Comment:
Commenting here so I know where to look later, this line does not need changing and actually it is breaking the whole classchange thing making it always set to "None"

Reply:
Do you mean this is something I break during reverting the reverts or something you did during your changes?

Reply:
Oh it is from the original author from the PRs